pkg/planner/core/find_best_task.go

```diff
@@ -264,8 +274,16 @@ func iteratePhysicalPlan4BaseLogical(
 	childTasks = childTasks[:0]
 	// The curCntPlan records the number of possible plans for selfPhysicalPlan
 	curCntPlan := int64(1)
+	var isParentPhyscicalHashJoin bool
+	if _, ok := selfPhysicalPlan.(*PhysicalHashJoin); ok {
+		isParentPhyscicalHashJoin = true
+	}
```

Comment:
check it here may not be a good idea. only mpp exchange will need it but it will pollute the whole physical property.

Reply:
Is there a structure that contains only MPP ？